cargo-apk/README.md

```diff
@@ -88,6 +88,11 @@ res = "path/to/res_folder"
 # If not specified, assets will not be included in the APK
 assets = "path/to/assets_folder"
 
+# Path to a folder containing a hierarchy of extra files to be added at the root of the APK.
+# All files will be added to the APK recursively.
+# If not specified, no extra files will be added to the APK
+extra_content = "path/to/extra_content_folder"
```

Comment:
I guess assets/resources are intended to be relative paths included in version control, while in your use case it's probably an os specific path. Also this doesn't help select the right library for the target architecture. Are there additional use cases for extra-content?

Reply:
Are you suggesting that I write the example as an absolute path?
In my particular use-case I do have the shared library file under my repository directory (although not included in version control), so I'm using a relative path.

Regarding selection of the right library for the target architecture, if the APK contains multiple architecture, I think what happens is that `dlopen()` already appropriately uses the correct set of paths (including the proper `lib` subfolders in the APK) depending on which architecture is used when running the app.

As for additional use cases, honestly, I can't think of any off the top of my head.

Reply:
I don't think we produce fat apk's because that can lead to very large apks. We could add the feature.

Reply:
@dvc94ch Apparently `cargo-apk` already produces fat apks when more target architectures are specified in `build_targets`. I don't know if separate APKs can be distributed anyway unless we dive into the new "app bundles" system.

Regarding assets/resources we have to be _very careful_ that those still travel the normal `aapt package` `-A`/`-S` as they need additional processing and are not "zipped" into the target APK verbatim.